src/components/projects/project-edit-form/project-edit-container/project-edit-details/project-edit-details.tsx

```diff
@@ -0,0 +1,163 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Project } from 'utils';
+import { Col, Container, Row, Form, InputGroup } from 'react-bootstrap';
+import PageBlock from '../../../../shared/page-block/page-block';
+import { wbsPipe, endDatePipe, fullNamePipe } from '../../../../../shared/pipes';
+import { WbsElementStatus } from 'utils/lib/types/project-types';
+
+// new parts added at the bottom
+interface projectDetailsProps {
+  project: Project;
+}
+
+const ProjectEditDetails: React.FC<projectDetailsProps> = ({ project }) => {
+  const statuses = Object.values(WbsElementStatus).filter((status) => status !== project.status);
+  const startDate =
+    project.workPackages.length > 0
+      ? project.workPackages
+          .reduce(
+            (min, cur) => (cur.startDate < min ? cur.startDate : min),
+            project.workPackages[0].startDate
+          )
+          .toLocaleDateString()
+      : 'n/a';
+  const endDate =
+    project.workPackages.length > 0
+      ? endDatePipe(
+          project.workPackages.reduce(
+            (min, cur) => (cur.startDate < min ? cur.startDate : min),
+            project.workPackages[0].startDate
+          ),
+          project.workPackages.reduce((tot, cur) => tot + cur.duration, 0)
+        )
+      : 'n/a';
+
+  const editDetailsInputBuilder = (
+    title: string,
+    type: string,
+    defaultValue: any,
+    prefix = '',
+    suffix = '',
+    placeholder = '',
+    readOnly = false
+  ) => {
+    const formInput = (
+      <Form.Group>
+        <InputGroup>
+          {prefix ? <InputGroup.Text>{prefix}</InputGroup.Text> : <></>}
+          <Form.Control
+            type={type}
+            defaultValue={defaultValue}
+            placeholder={placeholder}
+            readOnly={readOnly}
+          />
+          {suffix ? <InputGroup.Text>{suffix}</InputGroup.Text> : <></>}
+        </InputGroup>
+      </Form.Group>
+    );
+
+    return (
+      <>
+        <b>{title}</b>
+        {formInput}
+        <br />
+      </>
+    );
+  };
+
+  const statusSelect = (
+    <Form.Control as="select">
+      <option key={0} value={project.status}>
+        {project.status}
+      </option>
+      {statuses.map((status, index) => (
+        <option key={index + 1} value={status}>
+          {status}
+        </option>
+      ))}
+    </Form.Control>
+  );
+
+  const detailsBody = (
+    <Container fluid>
+      <Row>
+        <Col xs={12} md={6}>
+          {editDetailsInputBuilder('Project Name:', 'text', project.name)}
+          {editDetailsInputBuilder('WBS #:', 'text', wbsPipe(project.wbsNum))}
+          {editDetailsInputBuilder('Project Lead:', 'text', fullNamePipe(project.projectLead))}
+          {editDetailsInputBuilder(
+            'Project Manager:',
+            'text',
+            fullNamePipe(project.projectManager)
+          )}
```

Comment:
these need to be dropdown menus populated by the list of users

Reply:
do you want the dropdown to be in the same format as the status select dropdown?

Reply:
yeah that works